packages/mcfunction/test/parser/utils.ts

```diff
@@ -0,0 +1,34 @@
+import type { RootTreeNode } from '../../../java-edition/lib/dependency/mcmeta.js'
```

Comment:
the type of this changed from the `RootTreeNode` in `packages\mcfunction\src\tree\type.ts` to `packages\java-edition\src\dependency\mcmeta.ts`